doc/source/docs/user_guide/projections.rst

```diff
@@ -6,20 +6,19 @@
    import geopandas
 
 
-Managing Projections
-=========================================
+Projections
+===========
 
 
-Coordinate Reference Systems
+Coordinate reference systems
 -----------------------------
 
 The Coordinate Reference System (CRS) is important because the geometric shapes
 in a GeoSeries or GeoDataFrame object are simply a collection of coordinates in
 an arbitrary space. A CRS tells Python how those coordinates relate to places on
 the Earth.
 
-You can find the codes for most commonly used projections from
-`www.spatialreference.org <https://spatialreference.org/>`_.
+For reference codes in the most commonly used projections, see `Spatial Reference <https://spatialreference.org/>`_.
```

Comment:
In this specific case, I think a link text as "Spatial Reference" doesn't say much, while it is useful to know it is spatialreference.org we are referencing to.

(another thing is that we should probably stop referring to that site, since it is a bit outdated ..)

Reply:
Thoughts on external link icons? 
https://developers.google.com/style/cross-references#out-page

Reply:
> Thoughts on external link icons?

If that is something to be implemented, then it should be done in the `pydata-sphinx-theme`, not by adding icons to our code.